.github/workflows/node-flow-pull-request-checks.yaml

```diff
@@ -228,6 +228,44 @@ jobs:
       gradle-cache-password: ${{ secrets.GRADLE_CACHE_PASSWORD }}
       sonar-token: ${{ secrets.SONAR_TOKEN }}
 
+  hapi-tests-restart:
+    name: HAPI Tests (Restart)
+    uses: ./.github/workflows/node-zxc-compile-application-code.yaml
+    needs:
+      - build
```

Comment:
Please change to:
    needs:
      - dependency-check
      - spotless

Reply:
Fixed Jeffrey. Thanks